pkg/controller/migstage/stage.go

```diff
@@ -61,6 +61,22 @@ func (r *ReconcileMigStage) stage(stageMigration *migapi.MigStage) error {
 		PlanResources:   planResources,
 		BackupResources: stageResources,
 	}
+	// Mount propagation workaround
+	// TODO: Consider only bouncing restic pod if cluster version is 3.7-3.9
+	// This would require passing in cluster version to the controller, but might
+	// not be necessary
+	err, restartStatus := task.BounceResticPod()
+	if err != nil {
+		return err
+	}
+	// Update status if restart status has changed
+	if restartStatus != stageMigration.Status.ResticRestartStatus {
+		stageMigration.Status.ResticRestartStatus = restartStatus
+		err = r.Update(context.TODO(), stageMigration)
```

Comment:
Best I can tell, the result of this update failing is:
- The restic pods could get deleted more than necessary.
- The controller will list the pods more than necessary.

Right?

Reply:
Yes, because `BounceResticPod` will get called again with a stale status so it will essentially repeat the previous run.